lib/views/quiz/quiz_result.dart

```diff
@@ -73,11 +73,10 @@ class QuizResult extends StatelessWidget {
             textAlign: TextAlign.center,
           ),
         ),
-        Screen.width(context) >= 600
-            ? Heading(
-                "Quiz Completed!",
-              )
-            : Body("Quiz Completed!"),
+        Body(
+          "Quiz Completed!",
+          size: Screen.width(context) >= 600 ? 30 : 0,
```

Comment:
Create helper method Screen.isTablet(context)

Reply:
This has been done